### ISSUE TITLE
Healing Circle Transparency/Alpha

### ISSUE THREAD

**Issue by Drathamus** [MSRevive/MSCScripts#232]
Healing circle's transparency is not proper. The edges of the projected texture is visible while it is in use. Tested with dark staff circles as well as hoarfrost circles, and all their textures seem properly transparent. Healing circle is a rotating square.

It's hard to get a screen grab of it since it's so faint, but it's really noticeable on dark-textured floored maps.

https://steamuserimages-a.akamaihd.net/ugc/5102047032357643574/110E6198D10C2276729C9ADF23A3FDB064EF5AA4/?imw=5000&imh=5000&ima=fit&impolicy=Letterbox&imcolor=%23000000&letterbox=false

**Comment by Tsarenir** [comment]
Coincidentally I noticed this last night and went to fix it. I know how to fix it, and I did, the texture is set to additive but the black background of the holy symbol specifically isn't set to true black for some reason. However, when I compile the model, I get the attached result, and frankly I have no idea why... They don't animate at all, no rotation, just float as pictured.
![image](https://user-images.githubusercontent.com/39228913/210249734-11345b80-0d48-4148-acc8-4c506df58bdf.png)
